thorlcr/graph/thgraphslave.cpp

```diff
@@ -1492,6 +1492,7 @@ CJobSlave::CJobSlave(ISlaveWatchdog *_watchdog, IPropertyTree *_workUnitInfo, co
 {
     workUnitInfo.set(_workUnitInfo);
     workUnitInfo->getProp("token", token);
+    workUnitInfo->getProp("signature", signature);
```

Comment:
How / where is this signature being used?

Reply:
See the code I just added code to dasess.cpp , which will verify the signature before sending any permission requests to Dali. This gives us the same level of security as with the token, but perhaps we should verify the signature earlier, in case a workunit does not make Dali requests?

Reply:
See the code I just added to dasess.cpp, where i now verify the signature before sending any permission requests to Dali

Reply:
I can't see anywhere that 'signature' or CJobBase::querySignature() is used ... ?

Also it's unlikely that being in base+slave class is correct, as Thor slaves cannot interact with Dali.

So if this is used, the members/methods should be thgraphmaster.cpp only.

Reply:
Removed this call to get the signature